student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(id, age, classroom, name = 'Unknown', parent_permission: true)
+    super(id, age, name, parent_permission)
```

Comment:
- [ ] [Optional] If you add the recommendation above, you can delete `id` from lines 4 and 5.